A2rchi/interfaces/chat_app/app.py

```diff
@@ -197,3 +213,55 @@ def index(self):
     
     def terms(self):
         return render_template('terms.html')
+    
+    def like(self):
+        try:
+            # Get the JSON data from the request body
+            data = request.json
+
+            # Extract the HTML content and any other data you need
+            chat_content = data.get('content')
+            discussion_id = data.get('discussion_id')
+
+            feedback = {
+                "chat_content" :  chat_content,
+                "feedback"     :  "like",
+            }
+            ChatWrapper.update_or_add_discussion(self.data_path, "conversations_test.json", discussion_id, discussion_feedback = feedback)
+
+            response = {'message': 'Liked', 'content': chat_content}
+            return jsonify(response), 200
+
+
+        except Exception as e:
+            return jsonify({'error': str(e)}), 500
+        
+    def dislike(self):
+        try:
+            # Get the JSON data from the request body
+            data = request.json
+
+            # Extract the HTML content and any other data you need
+            chat_content = data.get('content')
+            discussion_id = data.get('discussion_id')
+            message = data.get('message')
+            incorrect = data.get('incorrect')
+            unhelpful = data.get('unhelpful')
+            inappropriate = data.get('inappropriate')
+
+            feedback = {
```

Comment:
Will make a longer comment in PR; but I think we may want to include a `message_id` here, which is unique within a discussion.